src/App.js

```diff
@@ -1,25 +1,91 @@
-import logo from './logo.svg';
 import './App.css';
+import contacts from "./contacts.json"
+import { useState } from 'react';
+
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  
+
+  const [contactInfo, setContacts] = useState(contacts.slice(0,5))
+
+  const randomArr = [...contacts].filter(contact => !contactInfo.includes(contact))
```

Comment:
I would write that piece of code inside of the `addRandomContact()` as you probably only use that one inside that function